(dp1
S'output'
p2
S"<type 'exceptions.ValueError'> invalid literal for long() with base 10: 'Utah'"
p3
sS'layer'
p4
S'/home/utahgroup/web2py/applications/welcome/controllers/ufopage.py'
p5
sS'code'
p6
S'<code object <module> at 0x7fa55d5c6bb0, file "/home/utahgroup/web2py/applications/welcome/controllers/ufopage.py", line 4>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/utahgroup/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/utahgroup/web2py/applications/welcome/controllers/ufopage.py", line 99, in <module>\n  File "/home/utahgroup/web2py/gluon/globals.py", line 421, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/utahgroup/web2py/applications/welcome/controllers/ufopage.py", line 36, in ufo\n    count = ufo_set.count()\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/objects.py", line 2267, in count\n    return db._adapter.count(self.query, distinct)\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/adapters/base.py", line 791, in count\n    self.execute(self._count(query, distinct))\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/adapters/base.py", line 779, in _count\n    sql_q = self.expand(query, query_env=query_env)\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/adapters/base.py", line 435, in _expand\n    rv = op(first, second, **optional_args)\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/dialects/base.py", line 344, in eq\n    self.expand(second, first.type, query_env=query_env))\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/dialects/__init__.py", line 96, in expand\n    return self.adapter.expand(*args, **kwargs)\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/adapters/base.py", line 445, in _expand\n    rv = self.represent(expression, field_type)\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/adapters/base.py", line 382, in represent\n    return super(SQLAdapter, self).represent(obj, field_type)\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/adapters/base.py", line 336, in represent\n    return self.representer.represent(obj, field_type)\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/representers/__init__.py", line 251, in represent\n    rv = self.get_representer_for_type(field_type)(rv, field_type)\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/representers/__init__.py", line 139, in __call__\n    return self.adapt(self.call(value, field_type))\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/representers/__init__.py", line 136, in _call\n    return self.inner_call(value)\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/representers/__init__.py", line 124, in _inner_call\n    return self.obj.f(self.representer, value, **kwargs)\n  File "/home/utahgroup/web2py/gluon/packages/dal/pydal/representers/base.py", line 26, in _id\n    return str(long(value))\nValueError: invalid literal for long() with base 10: \'Utah\'\n'
p11
s.